pkgs/development/coq-modules/equations/default.nix

```diff
@@ -6,6 +6,7 @@ with lib; mkCoqDerivation {
   repo = "Coq-Equations";
   inherit version;
   defaultVersion = switch coq.coq-version [
+    { case = "8.13"; out = "1.2.4+coq8.13"; }
     { case = "8.12"; out = "1.2.3+coq8.12"; }
     { case = "8.11"; out = "1.2.3+coq8.11"; }
```

Comment:
Version 1.2.4 is also available for Coq 8.11 and 8.12. Can you please also update those?

Reply:
ok